src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.iOSmacOS.cs

```diff
@@ -517,17 +560,17 @@ private class LayoutState : IEquatable<LayoutState>
 				BackgroundImage = backgroundImage;
 			}
 
-			public override int GetHashCode()
-				=> (Background?.GetHashCode() ?? 0 + BorderBrush?.GetHashCode() ?? 0) + (int)BackgroundSizing;
+			public override int GetHashCode() => new object[] { Bbox, Background, BackgroundSizing, BorderBrush, CornerRadius, BorderThickness, BackgroundImage }
```

Comment:
This looks expensive :) using a xors is likely to be enough.

Reply:
i wonder if the ValueTuple version be good enough?: (Bbox, Background,... ).GetHashCode()
or the anonymous class version?

Reply:
It could be that making a record of it would ease the writing.